.github/workflows/force-docker-build.yml

```diff
@@ -148,4 +148,15 @@ jobs:
             PIP_INSTALL_PERF=${{env.JINA_PIP_INSTALL_PERF}}
             PY_VERSION=${{matrix.py_version}}
             PIP_TAG=${{matrix.pip_tag}}
-          target: ${{env.BUILD_TARGET}}
\ No newline at end of file
+          target: ${{env.BUILD_TARGET}}
+
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: Create Release
+          token: ${{ secrets.JINA_DEV_BOT }}
+          inputs: '{ "release_token": "${{ env.release_token }}", "triggered_by": "TAG"}'
+        env:
+          release_token: ${{ secrets.JINA_CORE_RELEASE_TOKEN }}
```

Comment:
We run `Manual Docker Build` during `TAG` / `CD` & `MANUAL`. But `create-release` should only be triggered during `TAG`. That logic is missing here.

Reply:
Thanks a lot for your review @deepankarm. 

I thought that `if` statement would be enough to resolve it. Like 

```yaml 
if: startsWith(github.ref, 'refs/tags/v')
```

I just wanted to be sure before pushing things, what is your feedback on this ?

Reply:
Since `triggered_by` is passed in the input event, you can check `if: "${{ github.event.inputs.triggered_by }}" == "TAG"`. I'm not very sure of the syntax, please validate it before pushing the commit.

Reply:
Thanks a lot for the hint @deepankarm . 
I have checked the syntax by creating a dummy repo in my account and validated it. 
Also checked through Github documentation.

Reply:
@mrtrkmn The PR looks good. I'll prefer merging it sometime next week though as this might have some follow up actions :slightly_smiling_face:

Reply:
Great ! 
I am looking forward to see / contribute more on follow up actions or other parts of the repo :) 
Thanks a lot @deepankarm  :)